lib/matplotlib/colorbar.py

```diff
@@ -1274,6 +1298,111 @@ def _short_axis(self):
             return self.ax.xaxis
         return self.ax.yaxis
 
+    def _get_view(self):
+        """
```

Comment:
For all of the overridden methods here and below, I would just use `# docstring inherited` and move the additional colorbar-specific points into a comment (otherwise the rather long _set_view_from_bbox docstring can easily go out of sync).

Reply:
Done